examples/07_advanced/plot_beta_series.py

```diff
@@ -74,8 +74,9 @@
 data = fetch_language_localizer_demo_dataset(legacy_output=False)
 
 models, models_run_imgs, events_dfs, models_confounds = first_level_from_bids(
-    data.data_dir,
-    "languagelocalizer",
+    dataset_path=data.data_dir,
+    task_label="languagelocalizer",
+    sub_labels=["01"],
```

Comment:
mention parameters name and only run on the first subject since this is the only one we need